### PR TITLE
Negative margins for alignfull children of blocks with custom padding.

### PR DESCRIPTION
**Pull request by tellthemachines**
Trac ticket: https://core.trac.wordpress.org/ticket/61313

Syncs change from https://github.com/WordPress/gutenberg/pull/60716.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/block-supports/layout.php [hunk 1]
```diff
@@ -305,32 +305,32 @@ function wp_get_layout_style( $selector, $layout, $has_block_gap_support = false
...
 			);
+		}
 
-			if ( isset( $block_spacing ) ) {
-				$block_spacing_values = wp_style_engine_get_styles(
-					array(
-						'spacing' => $block_spacing,
-					)
-				);
+		if ( isset( $block_spacing ) ) {
```
Comment (ramonjd) [inline src/wp-includes/block-supports/layout.php:313]: Does this change deserve a `@since`?

**Review by ramonjd — APPROVED** [review]
I tested using the instructions over in https://github.com/WordPress/gutenberg/pull/60716

Things are working for me as described!

File: src/wp-includes/block-supports/layout.php (hunk 1 above)
Comment (tellthemachines) [reply src/wp-includes/block-supports/layout.php:313]: Hmm yeah that's a good question. Guess it's better to add one than not!

Comment (tellthemachines) [reply src/wp-includes/block-supports/layout.php:313]: Done.

**Comment by tellthemachines** [comment]
Committed in [r58243](https://core.trac.wordpress.org/changeset/58243).

**Pull request closed without merging**